recipes-bsp/tegra-binaries/tegra-redundant-boot_32.3.1.bb

```diff
@@ -20,9 +20,11 @@ do_install() {
 	install -m 0755 ${B}/usr/sbin/nvbootctrl ${D}${sbindir}
 	install -m 0755 ${B}/usr/sbin/nv_bootloader_payload_updater ${D}${sbindir}
 	install -m 0755 ${B}/usr/sbin/nv_update_engine ${D}${sbindir}
-	install -d ${D}${systemd_system_unitdir}
-	install -m 0644 ${B}/etc/systemd/system/nv_update_verifier.service ${D}${systemd_system_unitdir}
-	sed -i -e's,^After=nv\.service,After=nvstartup.service,' ${D}${systemd_system_unitdir}/nv_update_verifier.service
+	if ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)}; then
```

Comment:
This shouldn't be needed - systemd.bbclass should clean this up for us if systemd isn't being used.  What would be needed is the addition of an initscript equivalent for this systemd service... although I'm about at the point of dumping the initscripts completely and just making systemd a required distro feature.  NVIDIA stopped providing initscripts for their services when they started basing their BSP on Ubuntu 16.04, and I don't really have the bandwidth to maintain and test sysvinit-only builds.

Reply:
This is indeed not necessary. I've checked building without the change, and it builds fine for sysvinit.